src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -3045,9 +3046,23 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
     }
 
     isDisabledAxisLogCheckbox(vertical: boolean): boolean {
-        return vertical
+        const axisSelection = vertical
+            ? this.vertSelection
+            : this.horzSelection;
+
+        // dont disable the log box for rna_seq even if there are
```

Comment:
not sure if i understand this comment.  this method seems to always disable checkbox if there are negative numbers detected (hasNegativeNumbers)

Reply:
no its `!isValidMrnaProfile && hasNegativeNumbers` so if `isValidMrnaProfile` is true then it will not disable the checkbox